.changeset/loud-bees-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphcommerce/ecommerce-ui': minor
+---
+
+Export all react-hook-form-mui components
```

Comment:
Create a changeset for your changes for the `magento-newsletter: minor` and `magento-graphcms: patch` package